### PR TITLE
Add TK.getProjectResources

### PR DESCRIPTION
**Pull request by donlion**
Motivation
------------
Missing method

Design choices
------------
Followed existing patterns

Side-effects/other
------------

Tests
------------

Known issues/limitations
------------

Release dependencies
------------

Who should review it
------------
Someone from TimeKit

### CODE REVIEW

File: src/endpoints.js [hunk 1]
```diff
@@ -764,6 +764,24 @@ module.exports = function (TK) {
...
+   */
+  TK.getProjectResources = function(data) {
+
+    var id = data.id;
+    delete data.id;
+
+    return TK.makeRequest({
+      url: '/projects/' + id + '/resources',
+      method: 'get',
+      data: data
```
Comment (laander) [inline src/endpoints.js]: GET endpoint doesn't take any data in it's body, you can just delete this 👍
```suggestion
```

**Comment by donlion** [comment]
Ah, thanks! 🙏

File: src/endpoints.js [hunk 2]
```diff
@@ -764,6 +764,23 @@ module.exports = function (TK) {
...
+   * @return {Promise}
+   */
+  TK.getProjectResources = function(data) {
+
+    var id = data.id;
+    delete data.id;
+
+    return TK.makeRequest({
+      url: '/projects/' + id + '/resources',
+      method: 'get',
```
Comment (laander) [inline src/endpoints.js]: Remove hanging comma
```suggestion
      method: 'get'
```

**Comment by laander** [comment]
Done, released in `v1.18.1`

**Pull request merged**